src/portfft/descriptor.hpp

```diff
@@ -995,7 +996,7 @@ class committed_descriptor {
    */
   sycl::event compute_backward(scalar_type* inout_real, scalar_type* inout_imag,
                                const std::vector<sycl::event>& dependencies = {}) {
-    return compute_backward(inout_real, inout_real, inout_imag, inout_imag, dependencies);
+    return compute_backward(inout_real, inout_imag, inout_real, inout_imag, dependencies);
```

Comment:
I see you have changed from `real, real, imag, imag` to `real, imag, real, imag` in the compute_backward call only.  
Why did compute forward not require this change ?
Would current implementations that already support split_storage (workitem and subgroup) need some change on their end because of this as well ?

Reply:
This was a bugfix. Do you see arguments in wrong order in any other interfaces?